### PR TITLE
Fix tests for fixes to render menu to portal

### PR DESCRIPTION
**Pull request by develra**

### CODE REVIEW

File: src/base/index.spec.tsx [hunk 1]
```diff
@@ -252,20 +252,17 @@ describe('Portal', () => {
         <>
           <Portal />
           <Button onClick={() => setOpen(true)}>Open</Button>
-          {open && (
```
Comment (develra) [inline src/base/index.spec.tsx:255]: I've generally found conditionally rendering dialogs in RMWC to be pretty flaky on React 18 - in general I just let the control state decide if it should be visible or not, but happy to discuss!

File: src/base/portal.tsx [hunk 2]
```diff
@@ -49,6 +49,5 @@ export function PortalChild({
   if (portalEl) {
     return ReactDOM.createPortal(children, portalEl);
   }
-
-  return <>{children}</>;
+  return null;
```
Comment (develra) [inline src/base/portal.tsx:52]: Largest change is here - what should happen if portalEl is undefiend in PortalChild? I think just not rendering anything makes sense, but happy to discuss.

**Comment by EmiBemi** [comment]
I tried running the docs locally with the changes here. It fixes the problem with the portal rendering twice, but not the problem with the menu positioning... :(

**Comment by EmiBemi** [comment]
Okay, I managed to fix it combing some of the changes from this PR. Will merge and fix 👍

**Comment by EmiBemi** [comment]
Nvm.. I was running the wrong test. So we still do not have a fix, sadly.

**Pull request merged**